jphp-core/src/org/develnext/jphp/core/compiler/jvm/statement/expr/ForCompiler.java

```diff
@@ -37,11 +39,18 @@ public void write(ForStmtToken token) {
         LabelNode iter = new LabelNode();
         LabelNode end = new LabelNode();
 
-        expr.writeExpression(token.getCondition(), true, false);
-        expr.writePopBoolean();
+	for (Iterator<ExprStmtToken> i = token.getConditionExpr().iterator(); i.hasNext();) {
+	    ExprStmtToken expr = i.next();
+	    if (i.hasNext()) {
+		this.expr.writeExpression(expr, false, false);
+	    } else {
+		this.expr.writeExpression(expr, true, false);
+		this.expr.writePopBoolean();
```

Comment:
Is only the last expression will be checked?

```php
for ($i = 0, $x = 0; $i < 5, $x < 10; $i++, $x++) {
     echo "$i, $x\n";     
}
```
Please check this code and write a simple unit-test. What will be the result?